server/server.test.js

```diff
@@ -9,31 +11,16 @@ describe('server functional tests', () => {
     expect(theBestFlavor).toBe('grapefruit');
   });
 
-  test('Making a post request to / on the server should respond with "Hello World!"', () => {
+  test('Making a post request to / on the server should respond with "Hello World!"', async () => {
     // Making a post request to / means making a post request to the root of the server
-    // The root of the server basically means the "/" after localohst:3001
-    // This is similar to the line in app.js which does fetch('http://localhost:3001/',.. 
-    // this is cat ascii
-    //  /\_/\
-    // ( o.o ) // a heart 4 u!
-    //  > ^ <  //    ( V )
-    // (  -  ) //     \ /
-    //  | | |  //      v
-    //  | | |  //
-    //  | | |
-    //  | | |
-    //   V V
-    // What you want to do is make a post request with a package that you have installed. This
-    // might be the 'got' package.
-
-    // After you make the request, you should be able to get the response body and check that it
-    // is equal to "Hello World!"
-
-    // You can check things using assertions, eg `expect(theBestFlavor).toBe('grapefruit');`
-    // Make sure that you have the server started as another process while you're running the test,
-    // or the requests will go nowhere!
-
-    // Remember, npm run test-server to run this server, and npm run dev-server to run the app server
+    // The root of the server basically means the "/" after localhost:3001
+    // This is similar to the line in App.js which does fetch('http://localhost:3001/',.. 
+    await got.post('http://localhost:3001', {
+	    json: {
+		    "name": "Nishat",
+        "age": 28
+	    }
+    });
```

Comment:
heads up: you're not running an assertion here; you should assert that the response is what you expect. something like 

```
    const response = await got...
    expect(response).toBe('whatever you expect it tobe');
```

It likely _won't_ be a string. It might be a response object, and it might have a status code on it